tensorflow/core/kernels/data/dataset_test_base.cc

```diff
@@ -826,13 +829,14 @@ Status DatasetOpsTestBaseV2::MakeDatasetTensor(
 
   AttributeVector attributes;
   TF_RETURN_IF_ERROR(dataset_params.GetAttributes(&attributes));
-  gtl::InlinedVector<TensorValue, 4> inputs;
-  TF_RETURN_IF_ERROR(
-      dataset_params.GetInputs(input_datasets, created_tensors, &inputs));
-  std::vector<Tensor> input_tensors;
-  for (auto& tensor_value : inputs) {
-    input_tensors.emplace_back(*tensor_value.tensor);
+
+  std::vector<Tensor> dataset_inputs;
```

Comment:
this will end up holding non-dataset inputs, so I suggest renaming to just `inputs`

Reply:
Done.